services/credential-server-ui/src/components/inputOOBI/GetScannerButton.tsx

```diff
@@ -83,15 +87,28 @@ const GetScannerButton: React.FC<GetScannerButtonProps> = () => {
 
   return (
     <Container sx={{ py: 2 }}>
-      <Box sx={{ display: "flex", justifyContent: "right" }} mb={2}>
+      <Box sx={{ display: "flex", justifyContent: "left" }} mb={1}>
         <Button
-          variant="contained"
-          color="primary"
-          onClick={() => setShowInput(true)}
+          variant="text"
+          onClick={() => onBack()}
         >
-          Get Scanner
+          Back
```

Comment:
Can we make this back button variant "outlined" and have it beside the submit button (Back on the left, Submit on the right)